README.md

```
@@ -109,6 +110,45 @@ docker run -p 127.0.0.1:6274:6274 mcpjam/mcp-inspector
 
 The app is available at `http://127.0.0.1:6274`. Always use `-p 127.0.0.1:6274:6274` (not `-p 6274:6274`) to keep the inspector local-only. On macOS/Windows, connect to host MCP servers via `http://host.docker.internal:PORT` instead of `127.0.0.1`.
 
+# Coming from MCP Inspector?
+
+### 1. Two-sentence introduction
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)